src/cli/OutputGTFSZipCommand.ts

```diff
@@ -29,8 +29,12 @@ export class OutputGTFSZipCommand implements CLICommand {
     // when node tells you it's finished writing a file, it's lying.
     setTimeout(() => {
       console.log("Writing " + filename);
-      execSync(`zip -j ${filename} ${argv[3]}/*.txt`);
+      execSync(`zip -j '${escapeShellArg(filename)}' ${argv[3]}/*.txt`);
```

Comment:
should we also apply that to `argv[3]`?

Reply:
I have further updated the PR to not use escaping at all, as it didn't work on Windows.